mgv2/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM continuumio/miniconda
+
+COPY ./mgv2/ /mgv2
+
+
+RUN apt-get install -y python-dev
+RUN apt-get install -y gcc
+COPY ./clipper_admin/ /clipper_admin
+RUN pip install /clipper_admin
+
+RUN pip install requests\
+&& pip install -r /mgv2/requirements.txt
+
+ENV HOME /home
+
+COPY ./mgv2/config /home/.kube/
```

Comment:
Don't copy the config in. Check out [this issue](https://github.com/kubernetes-client/python/issues/254) for accessing K8s api from within a pod.